pandas/tests/frame/methods/test_drop_duplicates.py

```diff
@@ -425,3 +425,9 @@ def test_drop_duplicates_null_in_object_column(nulls_fixture):
     df = DataFrame([[1, nulls_fixture], [2, "a"]], dtype=object)
     result = df.drop_duplicates()
     tm.assert_frame_equal(result, df)
+
+
+def test_drop_duplicates_inplace_result():
+    df = DataFrame({"a": [1, 2, 3], "b": [1, 2, 4], "c": [1, 6, 5]})
+    result = df.drop_duplicates(inplace=True)
+    assert result is None
```

Comment:
can you use assert_frame_equal(df, expected) and construct the expected.

ideally this test goes with other basic drop_duplicates call and we parameterize on inplace

Reply:
yes. just checking the return value goes well with the existing 
`test_drop_duplicates_inplace`
moved it there.